src/ray/util/logging.cc

```diff
@@ -200,6 +200,7 @@ void RayLog::StartRayLog(const std::string &app_name, RayLogLevel severity_thres
     strftime(buffer, sizeof(buffer), "%Y%m%d-%H%M%S", localtime(&rawtime));
     std::string path = dir_ends_with_slash + app_name_without_path + "." + buffer + "." +
                        std::to_string(pid) + ".log";
+    stream_logger_singleton.out_.rdbuf()->pubsetbuf(0, 0);
```

Comment:
Does this have any performance implications?

Reply:
I don't know, but hopefully not significant ones. Though I think buffering isn't really appropriate in this layer regardless, since it can result in random log truncation. The kernel should already buffer sequential writes (which won't result in truncation), and [glog also has buffering support](/google/glog/blob/28321d8959574a8268de3ebe41d40cf65415fc9b/src/logging.cc#L145), so I think it's better to delegate to those.